<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1095</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Main window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView_processed">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>30</y>
      <width>275</width>
      <height>275</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>136</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_select">
       <property name="text">
        <string>Выбрать изображение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_bw">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Монохром</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_undo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_level">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Предел нормы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_level">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.250000000000000</double>
       </property>
       <property name="maximum">
        <double>0.900000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.900000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_zoom">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Зум</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_processed">
    <property name="geometry">
     <rect>
      <x>471</x>
      <y>0</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Обработанное изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_original">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>0</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Исходное изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recovered">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Восстановленное изображение</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_zoom">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>21</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_original">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>30</y>
      <width>275</width>
      <height>275</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_recovered">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>275</width>
      <height>275</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_size">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимальная площадь узла</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_amount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_amount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество секторов:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
